_docs-2/troubleshooting/tools.md

```diff
@@ -396,3 +397,39 @@ table.iterator.majc.vers=20,org.apache.accumulo.core.iterators.user.VersioningIt
 ...
 -----------------------------------------------
 ```
+
+## mode: print ACLs
```

Comment:
Following the above format, could emphasize the version it was added in the header

```suggestion
## mode: print ACLs (new in 2.1.1)
```